src/components/ChatRoom.tsx

```diff
@@ -0,0 +1,80 @@
+import { Show, For, createSignal, useContext, onMount } from "solid-js";
+import log from "loglevel";
+import { AppContextContext } from "../AppContextProvider";
+import { Group, Message, MessageParams } from "../types";
+import * as api from '../api'
+import "./ChatRoom.css"
+
+type ChatRoomProps = {
+  group: Group
+};
+
+
+export default function ChatRoom(props: ChatRoomProps) {
+  let divRef: HTMLDivElement | undefined
+  const [formValue, setFormValue] = createSignal('')
+  const [appState] = useContext(AppContextContext);
+  const [openChatRoom, setOpenChatRoom] = createSignal(true);
+  
+  onMount(() => { 
```

Comment:
If you use an effect, you won't need this:

```tsx
  createEffect(() => {
    if (props.group.chat) {
      divRef?.scrollIntoView();
    }
  });
```

Reply:
This will naturally then run scrollIntoView anytime there is an update to the chat or someone opens the chatRoom

Reply:
updated as you suggested